docs/managing-dependencies.md

```diff
@@ -145,7 +145,7 @@ When used together, `--without` takes precedence over `--with`. For example, the
 will only install the dependencies specified in the `test` group.
```

Comment:
I would suggest to make clear that the `test` group in this example must be an optional group. Maybe like this:

```suggestion
will only install the dependencies specified in the optional `test` group.
```